lib/puppet/pops/types/ruby_generator.rb

```diff
@@ -215,8 +215,10 @@ def class_body(obj, segments, bld)
     constants, others = obj.attributes(true).values.partition { |a| a.kind == PObjectType::ATTRIBUTE_KIND_CONSTANT }
     constants = constants.select { |ca| ca.container.equal?(obj) }
     unless constants.empty?
-      constants.each { |ca| bld << "\n  def self." << rname(ca.name) << "\n    _pcore_type['" << ca.name << "'].value\n  end\n" }
-      constants.each { |ca| bld << "\n  def " << rname(ca.name) << "\n    self.class." << ca.name << "\n  end\n" }
+      constants.each { |ca|
+        bld << "\n  def self." << rname(ca.name) << "\n    _pcore_type['" << ca.name << "'].value\n  end\n"
+        bld << "\n  def " << rname(ca.name) << "\n    self.class." << ca.name << "\n  end\n"
+      }
```

Comment:
I think the intent was to generate all of the class methods first, then all of the instance methods, which I think is preferable than what rubocop suggested

Reply:
Ok, I deleted the CombineableLoop changes and just moved the excludes out of the `rubocop_todo`.